x/ccv/utils/utils.go

```diff
@@ -78,3 +80,23 @@ func SendIBCPacket(
 	}
 	return nil
 }
+
+// ComputeConsumerUnbondingPeriod computes the unbonding period on the consumer
+// from the unbonding period on the provider (providerUnbondingPeriod)
+func ComputeConsumerUnbondingPeriod(providerUnbondingPeriod time.Duration) time.Duration {
```

Comment:
Why are we doing this instead of just using the same unbonding period?

Reply:
@jtremback was saying that it will help with the user experience on the provider to not have to wait longer for an undelegation to complete, especially since we cannot provide the exact info on how long users should wait. Most likely, the one day is sufficient to account for relaying delays, which means that in general the unbonding operations will complete first on the consumer chains.

Reply:
We should document this reasoning in the function go-doc